django/db/models/query.py

```diff
@@ -1153,6 +1172,36 @@ def __init__(self, *args, **kwargs):
         raise TypeError("EmptyQuerySet can't be instantiated")
 
 
+class RestrictedQuerySet(object):
+    # TODO: Ideas on how to make that nicer?
+    _allowed_methods = set()
```

Comment:
we should consider blacklisting versus whitelisting, disabling filtering is kind of a blacklist operation as it stands, rather than a whitelist